CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# CONTRIBUTING 
```

Comment:
Do we want to provide Contributing Guidelines?

Reply:
IMO, not in this PR, we should focus on getting project.json barebones running.